tests/acceptance/browzine-sersol-360-core-adapter-test.js

```diff
@@ -0,0 +1,67 @@
+describe("BrowZine SerSol 360 Core Adapter >", function() {
+  var serSol360Core = {}, searchResults = {};
```

Comment:
Can we `use strict` so we can use `let` and `const`?

Reply:
Good question. Actually took that out some time ago so we can keep the test environment as close to the production environment as possible.

Reply:
No problemo 👍